### ISSUE TITLE
sparse QR factorization - default value tol

### ISSUE THREAD

**Issue by KlausC** [JuliaLang/julia#26394]
There seems to be an error in the default value for the tolerance option in sparse QR factorization.

In Julia0.7 - SuiteSparse/src/spqr.jl:135ff we have:
```
# From SPQR manual p. 6
_default_tol(A::SparseMatrixCSC) =
      20*sum(size(A))*eps(real(eltype(A)))*maximum(norm(view(A, :, i))^2 for i in 1:size(A, 2))
```

While the document spqr_user_guide.pdf (version 2.0.6) on page 6 reads:

```
20*(m+n)*eps*sqrt(max(diag(A’*A))) where [m n]=size(A)
```

So the implementation is squared, what leads to unexpected scaling calamities.
The correct code should be:
```
# From spqr_user_guide.pdf  (version 2.0.6 - April 2016)  p. 6
_default_tol(A::SparseMatrixCSC) =
      20*sum(size(A))*eps(real(eltype(A)))*maximum(norm(view(A, :, i)) for i in 1:size(A, 2))
```

**Comment by Ramit-Pahwa** [comment]
Hi ,
I would like to try and solve the issue can you help me out , I am a new contributor to julia.

**Comment by andreasnoack** [comment]
Hi Ramit

This was actually fixed in #26429.
